public/clients/crpa.json

```diff
@@ -11,5 +11,261 @@
   "MATOMO": {"host": "https://stats.data.gouv.fr", "id": 181},
   "DISPLAY_NAME": "CRPA",
   "ALLOW_FEEDBACK": true,
-  "EXAMPLE_QUESTION": "Qu'est ce qu'un document administratif ?"
+  "EXAMPLE_QUESTION": "Qu'est ce qu'un document administratif ?",
+  "FAQ": {
```

Comment:
Copied from: https://github.com/etalab-ia/piaf-ml/blob/master/clients/crpa/knowledge_base/squad.json

The presence of this data automatically activates the FAQ